java-extras/src/main/java/org/triplea/java/DateTimeFormatterUtil.java

```diff
@@ -0,0 +1,29 @@
+package org.triplea.java;
+
+import com.google.common.annotations.VisibleForTesting;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
+import lombok.experimental.UtilityClass;
+
+/** Utility class for date-time formatting operations. */
+@UtilityClass
+public class DateTimeFormatterUtil {
```

Comment:
We already have TimeManager.java which has similar functionality.
Does it make sense to merge those 2 classes?

Reply:
Makes sense to combine them, yes.
`TimeManager` has a naming issue IMO. `Manager` is a bit of a smell in general, kingdom of nouns type deal and is generally not very meaningful. In this context it's not really managing state of any type. A final strike to `TimeManager` is I would not have thought to look for formatting tools in that class.

`DateTimeUtils` perhaps could be a better unified class name. Thoughts?

Reply:
Go with `DateTimeUtils` 🤷